bionic/decorators.py

```diff
@@ -59,15 +75,37 @@ def version(major=None, minor=None):
     minor: Integer or string (default 0)
         An arbitrary identifier for a function's nonfunctional characteristics.
 
+    ignore_bytecode: Boolean (default True)
+        Whether this entity's bytecode should be ignored.
+
+    suppress_bytecode_warnings: Boolean (default True)
+        Whether warnings from this entity's bytecode analysis should be ignored.
+
     Returns
     -------
     Function:
         A decorator which can be applied to an entity function.
     """
 
+    if ignore_bytecode is not None and not isinstance(ignore_bytecode, bool):
```

Comment:
Because that's what the other fields in version decorator are doing as well. I can move it here though.

Reply:
Ah, fair point. I see the other fields have converters because our internal code also creates CodeVersions. My main concerns is that we have to document the default behavior in the `@version` docstring, so it's a bit weird to have the actual transformation taking place elsewhere.